src/components/Dropdown/Dropdown.tsx

```diff
@@ -953,7 +959,8 @@ const Dropdown = ({
           ...(valueContainerProps ? valueContainerProps.containerProps : {}),
         }}
       >
-        {searchCharacterCount === 0 && (!values || !values.length) && !focusWithin && (
+        {((searchCharacterCount === 0 && (!values || !values.length) && !focusWithin) ||
+          (!showSelectedValues && !focusWithin)) && (
```

Comment:
again if you can reduce the growing craziness here with some variables that would be awesome

Reply:
Looking into it now!